packages/headless-react/package.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Almost everything here is copied from `@coveo/atomic-react`, I basically just removed some unneeded dependencies.